RecoLocalCalo/EcalRecAlgos/BuildFile.xml

```diff
@@ -14,7 +14,7 @@
 
 <use   name="eigen"/>
 
-<Flags CXXFLAGS="-Ofast"/>
+<flags CXXFLAGS="-O3"/>
```

Comment:
Because according to the measurements, it turned out to be faster.

I did not try to use `-Ofast` here and `-O3` in the EcalRecProducers package; I have only tried `-Ofast` / `-O2` vs `-O3` / `-O3` .